src/chocolatey/infrastructure.app/domain/SourceType.cs

```diff
@@ -26,7 +26,7 @@ public enum SourceType
     {
         //this is what it should be when it's not set
         normal,
-
+        [Obsolete("The WebPI service has been retired, and this source type will be removed in v2.0.0 of Chocolatey.")]

```

Comment:
You don't need to have this, the entire enumeration is already deprecated in favor of the `SourceTypes` class instead. Although, there should not be any harm in keeping it though.

Reply:
Figured I'd put it in for consistency to avoid confusion for now yeah.